src/main/webapp/app/overview/student-questions/student-question/student-question.component.html

```diff
@@ -0,0 +1,81 @@
+<!-- question not in edit mode -->
+<ng-container *ngIf="!isEditMode">
+    <div class="row mb-1">
+        <div class="col px-0">
+            <span>{{ 'artemisApp.courseOverview.exerciseDetails.faq.questionLabel' | translate }}</span>
+        </div>
+        <div class="col-auto px-0">
+            <fa-icon
+                class="clickable mr-1"
+                *ngIf="isQuestionAuthor || isAtLeastTutorInCourse"
+                [ngbTooltip]="'artemisApp.courseOverview.exerciseDetails.faq.editQuestion' | translate"
+                container="body"
+                [icon]="'pencil-alt'"
+                (click)="$event.stopPropagation(); isEditMode = true"
+            ></fa-icon>
+            <jhi-confirm-icon
+                class="clickable mr-1"
+                *ngIf="isQuestionAuthor || isAtLeastTutorInCourse"
+                (click)="$event.stopPropagation()"
```

Comment:
same here

Reply:
I removed them all. I must have missed that during the refactoring.